ui/src/app/components/help-sidebar.tsx

```diff
@@ -206,6 +210,7 @@ interface Props {
   setSidebarState: Function;
   shareFunction: Function;
   sidebarOpen: boolean;
+  notebookStyles: boolean;
```

Comment:
this could probably be better named - taking suggestions!

Reply:
Hmm all i can think of  is overrideForNotebook or isNotebook